test/unit/concerns/background_deletion_test.rb

```diff
@@ -36,5 +36,19 @@ def test_delete_integration
       DeleteObjectHierarchyWorker.expects(:perform_later).with(user, anything, 'destroy').never
       DeleteObjectHierarchyWorker.perform_now(double_object)
     end
+
+    test 'all associations of the config can be constantized' do
+      Rails.application.eager_load!
+
+      ActiveRecord::Base.descendants.each do |klass|
+        next if [DoubleObject, DoubleActiveRecordObject, SecondDoubleActiveRecordObject].include?(klass) || !klass.respond_to?(:background_deletion)
```

Comment:
Please read these 2 comments/threads:
- https://github.com/3scale/porta/pull/1685#discussion_r380169618
- https://github.com/3scale/porta/pull/1685#discussion_r380170686

Btw I ended up doing `!klass.respond_to?(:background_deletion)` instead of `klass.background_deletion.blank?` for this:
![image](https://user-images.githubusercontent.com/11318903/74657017-a9d98700-518f-11ea-81b9-90b1883e1258.png)

Reply:
You can use `ApplicationRecord.descendants` instead of using ActiveRecord::Base 🤕

Reply:
Yes OK only one loop using `ApplicationRecord.descendants`  👍

Reply:
Done :sunglasses: :dancer: